src/enabler.cpp

```diff
@@ -21,48 +33,63 @@ void Enabler::enableAutostart()
 void Enabler::disableAutostart()
 {
     qDebug() << "disable autostart called";
-    QFile::copy(":/desktop-file/org.opensuse.opensuse_welcome.desktop", QDir::homePath() + "/.config/autostart/org.opensuse.opensuse_welcome.desktop");
+    QFile::copy(":/desktop-file/org.opensuse.opensuse_welcome.desktop",
+                QDir::homePath() + "/.config/autostart/org.opensuse.opensuse_welcome.desktop");
     qDebug() << QDir::homePath() + "/.config/autostart/org.opensuse.opensuse_welcome.desktop";
 }
 bool Enabler::fileExists(QString path)
```

Comment:
This can just be `QFile::exists(path)`. The case that the path is a non-file shouldn't be relevant.